tap_zendesk/discover.py

```diff
@@ -58,10 +58,18 @@ def discover_streams(client, config):
         streams.append({'stream': stream.name, 'tap_stream_id': stream.name, 'schema': schema, 'metadata': stream.load_metadata()})
 
     if error_list:
+
+        total_stream = len(STREAMS.values()) # Total no of streams
         streams_name = ", ".join(error_list)
-        message = "HTTP-error-code: 403, Error: You are missing the following required scopes: read. "\
-                    "The account credentials supplied do not have read access for the following stream(s):  {}".format(streams_name)
-        raise ZendeskForbiddenError(message)
+        message = "The account credentials supplied do not have read access for the following stream(s): {}."\
+            "The data for the mentioned streams will not be collected if selected due to a lack of 'read' permission.".format(streams_name)
```

Comment:
@prijendev Update the message to,
"'The account credentials supplied do not have 'read' access to the following stream(s): {}. The data for these streams would not be collected due to lack of required permission"

Reply:
Updated message